source/_integrations/prometheus.markdown

```diff
@@ -122,6 +122,8 @@ prometheus:
 
 {% include common-tasks/filters.md %}
 
+> Note: Metrics are exported only for certain domains. For details, see the `def _handle_` functions in the [Prometheus integration source code](https://github.com/home-assistant/core/blob/dev/homeassistant/components/prometheus/__init__.py).
```

Comment:
- These are GitHub syntax notes, which isn't supported by our website. Please make sure you test a PR before submitting it.
- We should send end-users into code; if there is information on the linked code that is relevant, we should add it here in normal text.

Reply:
ok, done

Reply:
I've removed the code links and netlify preview looks good now